minemeld/flask/sns.py

```diff
@@ -0,0 +1,105 @@
+import requests
+import json
+import os.path
+import uuid
+import minemeld
+from .logger import LOG
+from . import config
+
+SNSAPIURL = config.get('SNS_URL', 'https://o11kiw0vpe.execute-api.us-east-1.amazonaws.com/run')
```

Comment:
We should change the default value for the URL to something stable

Reply:
Sure. The API GW is hosted in Palo Alto Networks AWS account. Could you, please, provide a panw.io mapping for the url?